docs/getting_started.md

```diff
@@ -209,23 +210,8 @@ Generate the cluster.yaml for workload cluster
 
 Inspect and make any changes
 ```shell
-# for vms as byohosts
-$ BUNDLE_LOOKUP_TAG=v1.23.5 CONTROL_PLANE_ENDPOINT_IP=10.10.10.10 clusterctl generate cluster byoh-cluster \
```

Comment:
oops. Didn't know I was replicating this. Thanks.
But you should probably remove the other set of commands. Ideally, we want `BUNDLE_LOOKUP_TAG` to be provided as part of the command.

Reply:
Oh. I missed the `BUNDLE_LOOKUP_TAG`, but why are we hardcoding this value? Shouldn't we extract it from somewhere?

Reply:
`BUNDLE_LOOKUP_TAG` is the `BYOH` bundle OCI Image tag and will be the same as the k8s version. This might be removed going forward.

Reply:
So should I include `BUNDLE_LOOKUP_TAG` for now or just let it be?

Reply:
Yup, please include. The cluster-template expects `BUNDLE_LOOKUP_TAG` to be present. We will clean it up in a future PR.